docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
@lextiz If you're using other promise-supporting libraries that use Bluebird, you will most likely derive promises from them, e.g.

```js
function myFunctionality(blah) {
  return dbLibrary.queryReturningPromise(query).then(doSomeProcessing);
}
```

Originally Bluebird used non-global events, but a bunch of issues were quickly opened that stemmed from the above, so as a result it now has [global events](http://bluebirdjs.com/docs/api/error-management-configuration.html#global-rejection-events)